test-apps/appui-test-app/standalone/package.json

```diff
@@ -6,9 +6,10 @@
   "version": "0.0.0",
   "homepage": "http://localhost:3000/",
   "scripts": {
-    "build": "npm run -s build:backend && npm run -s build:frontend",
+    "build": "npm run -s build:backend && npm run -s frontend:tsc",
     "build:backend": "tsc -p tsconfig.backend.json",
     "build:frontend": "cross-env DISABLE_NEW_JSX_TRANSFORM=true SKIP_PREFLIGHT_CHECK=true DISABLE_ESLINT_PLUGIN=true TRANSPILE_DEPS=false USE_FAST_SASS=true react-scripts --max_old_space_size=8192 build",
+    "frontend:tsc": "tsc",
```

Comment:
Why do we need this now?

Reply:
Previously used react-script build was running a webpack which takes a long time to bundle the app, which we are not really using. It should be enough to run the tsc to catch any errors (since the e2e tests run the test-app devserver either way).